pkg/telemetry/promql.go

```diff
@@ -70,6 +70,18 @@ var promqlStats = []promqlTelemetry{
 	}, {
 		name:  "promql_query_execution_time_p99",
 		query: "histogram_quantile(0.99, sum by(le) (rate(promscale_metrics_query_duration_seconds_bucket[1h])))",
+	}, {
+		name:  "promql_query_remote_read_batch_execution_time_p50",
+		query: "histogram_quantile(0.5, sum by(le) (rate(promscale_metrics_query_remote_read_batch_duration_seconds_bucket[1h])))",
```

Comment:
```suggestion
		query: "histogram_quantile(0.5, sum by(le) (rate(promscale_query_remote_read_batch_duration_seconds_bucket[1h])))",
```

This is the query added in PR. Same for the ones below.

Reply:
otherwise, add metrics as namespace

Reply:
Maybe the source Prometheus metric should be renames to `promscale_metrics_query_remote_read_batch_duration_seconds_bucket` ? Or does that metric also count non-metric remote read queries?

Reply:
added metrics as subsystem